src/qz/printer/status/CupsStatusHandler.java

```diff
@@ -21,7 +22,9 @@
 
     private static Cups cups = Cups.INSTANCE;
     private int lastEventNumber = 0;
-    private HashMap<String, ArrayList<String>> lastPrinterStatusMap = new HashMap<>();
+    //todo could perhaps be the same list
+    private HashMap<String, ArrayList<Status>> lastPrinterStatusMap = new HashMap<>();
+    private HashMap<String, ArrayList<Status>> lastJobStatusMap = new HashMap<>();
 
     public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
```

Comment:
Will `handle` need to be synchronized now, to avoid concurrent modifications exceptions?